.goreleaser.yml

```diff
@@ -34,6 +34,10 @@ archives:
     amd64: x86_64
 checksum:
   name_template: 'checksums.txt'
+signs:
+  - cmd: cosign
+    args: ["sign-blob", "--oidc-issuer=https://token.actions.githubusercontent.com", "--output=${signature}", "${artifact}"]
```

Comment:
This part is missing the `signature: ${artifact}.sig` value.. also it might makes sense to wait for the goreleaser support for releasing certificates, too :)

Reply:
IMHO we don't need this one because we're using `artifacts: checksum,` so it creates `checksums.txt.sig` by default, as you can see from [here](https://github.com/Dentrax/ko/releases/download/v0.9.3-signoidc/checksums.txt.sig). Also, we don't need to wait to release the certificates feature because in cosign with https://github.com/sigstore/cosign/pull/991, we can verify blobs in keyless mode.  The problem here is that these certificates are short-lived IIRC, so for users who might want to use these to verify the ko binary might be problematic because these certificates will be expired in a short time, WDYT @shibumi @imjasonh?

Reply:
Just for the record (@developer-guy  know this already). Short-Lived certificates are not problematic, because it just refers to the signature generation, not the signature verification. See also: https://github.com/goreleaser/goreleaser/issues/2659#issuecomment-965756308